.gitignore

```diff
@@ -1,9 +1,7 @@
-# .NET
-# .NET Core
-# .NET SDK
 .DS_Store
 .project
 .vscode
 appsettings.json
 bin/
 obj/
+HolyshiftService/Properties
```

Comment:
You actually need this folder don't git ignore it. This folder holds the config that will turn on the dev service when you launch the service.
> This is specific to launch options not in-app config. (e.g. Environment Variables)